plugins/module_utils/managed_cluster_addons/__init__.py

```diff
@@ -0,0 +1,15 @@
+from inspect import isclass
+from pkgutil import iter_modules
+from pathlib import Path
+from importlib import import_module
+
+# iterate through the modules in the current package
+package_dir = Path(__file__).resolve().parent
```

Comment:
this looks so cool 👍

Reply:
https://docs.python.org/3/tutorial/modules.html#importing-from-a-package
not sure if this will help us to support '*'. so adding the module names into `__all__` ? so it only need to have all file names without .py I guess ❓

Reply:
The sanity check does not allow non-empty __init__.py and throws this errors 😭 
```
ERROR: Found 1 empty-init issue(s) which need to be resolved:
ERROR: plugins/module_utils/managed_cluster_addons/__init__.py:0:0: empty __init__.py required
```

Reply:
To fix sanity check error, I have to use empty `__init__.py`